### PR TITLE
Try capturing logs from GH actions

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Review by ezr-ondrej — APPROVED** [review]

**Comment by ezr-ondrej** [comment]
Do you want to get this in, or did you want them just once?

**Comment by adamruzicka** [comment]
There were some test failures that I wanted to look into. Luckily they happen here as well, so I'd keep this one as a playground and then merge it together with the "real" fix

File: test/unit/job_invocation_composer_test.rb [hunk 1]
```diff
@@ -925,12 +925,14 @@ def provider_inputs
...
-        )
+        Foreman.settings._add('remote_execution_form_job_template',
+          context: :test,
+          type: :string,
+          category: 'Setting::RemoteExecution',
+          full_name: '',
+          description: '',
+          default: '')
+        Setting[:remote_execution_form_job_template] = setting_template.name
       end
```
Comment (adamruzicka) [inline test/unit/job_invocation_composer_test.rb:936]: @ezr-ondrej is this how settings in tests are supposed to be handled?

File: test/unit/job_invocation_composer_test.rb [hunk 2]
```diff
@@ -925,12 +925,14 @@ def provider_inputs
...
-          :category => 'Setting::RemoteExecution',
-          :value => setting_template.name
-        )
+        Foreman.settings._add('remote_execution_form_job_template',
+          context: :test,
+          type: :string,
+          category: 'Setting::RemoteExecution',
+          full_name: '',
+          description: '',
+          default: '')
```
Comment (ezr-ondrej) [inline test/unit/job_invocation_composer_test.rb:934]: this should not be needed, that should be already there, only settings that are added only in tests need to use this.
Simple `Setting[<name>] = value` should do the job for existing settings.

File: test/unit/job_invocation_composer_test.rb (hunk 1 above)
Comment (ezr-ondrej) [reply test/unit/job_invocation_composer_test.rb:936]: ^^

**Comment by adamruzicka** [comment]
:green_apple:

**Comment by ezr-ondrej** [comment]
Thanks @adamruzicka ! :)

**Pull request merged**